packages/devextreme/js/__internal/ui/collection/collection_widget.edit.ts

```diff
@@ -479,7 +485,9 @@ class CollectionWidget<
 
         const { grouped } = this.option();
 
+        // @ts-expect-error items inside TItem
```

Comment:
we can use type guard here, smth like 
```suggestion
        const hasSubItems = (item: TItem): item is TItem & { items: TItem[] } => isObject(item) && 'items' in item && Array.isArray(item.items);

        if (grouped && hasSubItems(normalizedSelection)) {
```

Reply:
fixed in https://github.com/DevExpress/DevExtreme/pull/30768/commits/bac2839415ee638ae839913f6cb7856a12d253e5